resources/assets/components/Campaign/index.js

```diff
@@ -101,6 +102,13 @@ const Campaign = ({ id }) => {
         <h4>Campaign ID</h4>
         <p>{campaign.id}</p>
 
+        <h4>Contentful Campaign ID</h4>
```

Comment:
do you think the campaigns team will know where to pull this ID from in contentful? or would it be helpful to add some text to clarify that.

Reply:
@ngjo is [working on some](https://github.com/DoSomething/rogue/pull/994#issuecomment-591630678) that we'll add it!

Reply:
oh dang i didn't even read all the way through that comment! 💃